db/pf-schema-X.Y.Z.sql

```diff
@@ -1261,3 +1255,21 @@ CREATE TABLE auth_log (
   KEY pid (pid),
   KEY  attempted_at (attempted_at)
 ) ENGINE=InnoDB;
+
+--
+-- Creating chi_cache table
+--
+
+CREATE TABLE `chi_cache` (
+  `key` VARCHAR(767),
+  `value` LONGTEXT,
+  `expires_at` REAL,
```

Comment:
I would suggest to use DATETIME.
So we can use a single source of when to expire of when to expire an entry instead of the individual clients.